### PR TITLE
Improve email validation

### PR DESCRIPTION
**Pull request by TatuArvela**

### CODE REVIEW

File: src/utils/formValidation.test.ts [hunk 1]
```diff
@@ -15,6 +15,16 @@ describe('formValidation', () => {
     });
   });
 
+  describe('mustBeEmail', () => {
+    test('should return undefined if the email is valid', () => {
+      expect(mustBeEmail('test@example.com')).toBeUndefined();
+    });
+
+    test('should return an error message if the email is invalid', () => {
+      expect(mustBeEmail('test@.com')).toMatchSnapshot();
```
Comment (santtul) [inline src/utils/formValidation.test.ts:24]: I think we should start relying less on snapshots if possible:
- It's tempting to tell jest to just update all snapshots when something fails -> we might miss problems
- You cannot see the actual "contract" or expected values from the unit test -> harder to understand the function/module under test and the unit test doesn't server as documentation of the contract or API anymore

**Review by santtul — APPROVED** [review]
Approved, but added one comment to think about.

File: src/utils/formValidation.test.ts (hunk 1 above)
Comment (TatuArvela) [reply src/utils/formValidation.test.ts]: 👍 Not strictly in the scope of this PR, but changed them anyway

Comment (santtul) [reply src/utils/formValidation.test.ts]: Good 👍

**Pull request merged**